### PR TITLE
Added verbose flag to /sf timings

### PR DESCRIPTION
**Pull request by TheBusyBiscuit**
## Description

@WalshyDev wanted this.
Also finally added locale to the /sf timings command. Added an abstract framework to allow for future flags too.

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/TimingsCommand.java [hunk 1]
```diff
@@ -15,27 +21,76 @@
...
 
+    @ParametersAreNonnullByDefault
+    private boolean hasInvalidFlags(CommandSender sender, String[] args) {
+        boolean hasInvalidFlags = false;
+
+        // We start at 1 because args[0] will be "timings".
+        for (int i = 1; i < args.length; i++) {
+            String argument = args[i].toLowerCase(Locale.ROOT);
+
+            if (!argument.startsWith(FLAG_PREFIX) || !flags.contains(argument.substring(2))) {
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/TimingsCommand.java:67]: This should be startsWith not doesnt. We shouldn't be firing "invalid flag" for a letter a.

File: src/main/resources/languages/messages_en.yml [hunk 2]
```diff
@@ -27,6 +26,12 @@ commands:
     charge-success: Item has been charged!
     not-rechargeable: This item can not be charged!
 
+  timings:
+    description: Lag-Info about your Server
```
Comment (WalshyDev) [inline src/main/resources/languages/messages_en.yml:30]: "Lag-Info" ew

"Timing info for Slimefun and it's addons." Would be nicer

**Review by WalshyDev — APPROVED** [review]
LGTM. Just a little nitpick thing

**Review by WalshyDev — APPROVED** [review]
LGTM

**Review by svr333 — APPROVED** [review]
Looks good

**Pull request merged**